Hardware.h

```diff
@@ -1,14 +1,41 @@
 #pragma once
-
-#include "Encoder.h"
+#include "Direction.h"
 #include "Led.h"
 #include "Motor.h"
 #include "RangeFinder.h"
 
+enum Relation { LEFT, FRONT, RIGHT };
+
 class Hardware {
   public:
+    // PIN DEFINITIONS
+    // static const unsigned MOTOR_LEFT_PIN = 17
+    // ...
+    // ...
+    // static const unsigned RANGE_FINDER_FRONT_PIN = 18
+
+    static const unsigned NUM_RANGE_FINDERS = 3;
+
     Hardware();
-    ~Hardware();
+
+    void moveForward(unsigned mm);
+    void rotate(int deg);
+
```

Comment:
Return a bool value to check whether the robot encountered an obstacle.

Also how would we allow the Mouse class to know that the robot has moved entirely from one cell to another?